apollo-converters/pojo/src/main/java/com/apollographql/android/converter/pojo/ApolloConverterFactory.java

```diff
@@ -2,25 +2,38 @@
 
 import com.apollographql.android.api.graphql.Operation;
 import com.apollographql.android.api.graphql.Response;
+import com.apollographql.android.api.graphql.TypeMapping;
 
 import java.lang.annotation.Annotation;
 import java.lang.reflect.ParameterizedType;
 import java.lang.reflect.Type;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
 
 import okhttp3.ResponseBody;
 import retrofit2.Converter;
 import retrofit2.Retrofit;
 
 /** TODO */
 public class ApolloConverterFactory extends Converter.Factory {
+  private final Map<TypeMapping, CustomTypeAdapter> customTypeAdapters = new HashMap<>();
```

Comment:
nitpick: https://publicobject.com/2016/02/08/linkedhashmap-is-always-better-than-hashmap/